lib/src/bridge/bluetooth_state_mixin.dart

```diff
@@ -37,7 +39,7 @@ mixin BluetoothStateMixin on FlutterBLE {
     yield* _adapterStateChanges.map(_mapToBluetoothState);
   }
 
-  BluetoothState _mapToBluetoothState(dynamic rawValue) {
+  BluetoothState _mapToBluetoothState(String? rawValue) {
```

Comment:
Unless Dart complains, this should be typed as non-nullable. If for any reason a null came from the native side, then it's an error and should not be swept under the rug.